submodules/wizard/wizard.js

```diff
@@ -547,7 +547,12 @@ define(function(require) {
 				$form = $template.find('form'),
 				// No need to validate if step won't be completed yet
 				isValid = !eventArgs.completeStep || monster.ui.valid($form),
-				accountContactsData;
+				accountContactsData,
+				arePropertiesEmpty = function(data) {
+					if (_.every(data, _.isEmpty)) {
+						_.merge(data, { isEmpty: true });
```

Comment:
In this case `lodash#merge` is not necessary. Since only one property is set into the `data` object, it can be done directly:
```suggestion
						data.isEmpty = true;
```

Reply:
Sorry to barge in but in this case the `if` statement is superfluous and can be avoided with:
```js
data.isEmpty = _.every(data, _.isEmpty);
```
You don't have to apply this change, just pointing it out for future reference.